### ISSUE TITLE
Blank Dashboard

### ISSUE THREAD

**Issue by rchac** [LibreQoE/LibreQoS#206]
After logging in the dashboard displays no live data, although it is possible to click on individual circuits from Shaped Devices and see live throughput.

![image](https://user-images.githubusercontent.com/22501920/213863491-11021a8e-62c9-4474-b62f-834d0503bd70.png)
![image](https://user-images.githubusercontent.com/22501920/213863613-583a66dd-37da-4a81-b21f-560d14b17cbe.png)

Then, after a few hours, a throughput spike displays, but does not go away.
![image](https://user-images.githubusercontent.com/22501920/213874297-ff5a3f05-d6fd-4991-aa7c-467f1260a52e.png)

**Comment by rchac** [comment]
![image](https://user-images.githubusercontent.com/22501920/213908040-9b70e9d0-d5d3-4487-a5ac-882156db72c0.png)
Seems to be an error in src/rust/lqos_queue_tracker/src/queue_structure/queue_node.rs ? I think the to_flat function may have a bug.

**Comment by gbnpl** [comment]
Is this bug correct in current main branch ?

**Comment by rchac** [comment]
Bug exists in main. It seems that it only affects instances where network.json has nested nodes. So flat networks should not be affected by this bug.

**Comment by gbnpl** [comment]
We have a flat network and this bug also occurs.
I'm thinking about if it matters which daemon is started first lqosd or lqos_node_manager

**Comment by rchac** [comment]
In the terminal screen window for lqos_node_manager, what errors display?

**Comment by gbnpl** [comment]
Jan 21 11:16:30 ubntlibreqos lqos_node_manager[97724]: Error: Error reading CSV record: Err(Error(UnequalLengths { pos: Some(Position { byte: 139072, line: 2284, record: 2283 }), expected_len: 1, len: 13 }))
Jan 21 11:16:30 ubntlibreqos lqos_node_manager[97724]: Error: Error reading CSV record: Err(Error(UnequalLengths { pos: Some(Position { byte: 139134, line: 2285, record: 2284 }), expected_len: 1, len: 13 }))
Jan 21 11:16:30 ubntlibreqos lqos_node_manager[97724]: Error: Error reading CSV record: Err(Error(UnequalLengths { pos: Some(Position { byte: 139196, line: 2286, record: 2285 }), expected_len: 1, len: 13 }))
Jan 21 11:16:30 ubntlibreqos lqos_node_manager[97724]: ShapedDevices.csv changed
Jan 21 11:16:36 ubntlibreqos lqos_node_manager[97724]: ShapedDevices.csv changed

**Comment by rchac** [comment]
When I started my box with the latest pull, I noticed the dashboard was working, albeit with no real data going through. This was while the customer data was blocked (interface disabled on edge side). Then, as soon as I enabled the interface and allowed traffic to pass through into the LibreQoS box, I saw a brief spike of traffic where it looked like it was going to work. Right before it could update the top 10 list, it stopped at that point and froze up.
![image](https://user-images.githubusercontent.com/22501920/215330550-7d72af31-1834-4d85-909d-1793541bf24e.png)

I think something about actually displaying the Shaped Devices on the top 10 lists is what breaks things.

Running ```cat /var/log/syslog | grep lqos``` yields:
![image](https://user-images.githubusercontent.com/22501920/215330630-96c0b3fd-7232-4efb-845f-1b83a703ada4.png)
